kore/test/Test/Kore/Equation/Simplification.hs

```diff
@@ -0,0 +1,100 @@
+module Test.Kore.Equation.Simplification
+    ( test_simplifyEquation
+    ) where
+
+import Prelude.Kore
+
+import Kore.Equation.Equation
+import Kore.Equation.Simplification
+import Kore.Internal.MultiAnd
+    ( MultiAnd
+    )
+import qualified Kore.Internal.MultiAnd as MultiAnd
+import Kore.Internal.TermLike
+
+import Test.Kore.Equation.Common
+    ( functionAxiomUnification_
+    , sortR
+    )
+import qualified Test.Kore.Step.MockSymbols as Mock
+import Test.Kore.Step.Simplification
+    ( runSimplifier
+    )
+
+import Test.Tasty
+import Test.Tasty.HUnit.Ext
+
+test_simplifyEquation :: [TestTree]
+test_simplifyEquation =
+    [ testGroup "Unification equations: in all cases the argument gets\
+                \ simplified and substituted into the left term; the argument\
+                \ is then removed."
```

Comment:
I think this would be better as a comment, so that it does not take up so much space in the test output.

```suggestion
    [ testGroup "Unify arguments"
        -- In all cases the argument gets simplified and substituted into the left term;
        -- the argument is then removed.
```